src/digitaltwin/tables.py

```diff
@@ -81,6 +81,42 @@ class UserLogInfo(Base):
     geometry = Column(Geometry("POLYGON", srid=2193))
 
 
+class MedusaScenarios(Base):
+    """
+    Class representing the 'MedusaUserLogInfo' table.
+
```

Comment:
It would be good to place this new code in `pollution_tables.py` instead, because it would make it easier to pull out later